etc/openqa/workers.ini

```diff
@@ -14,7 +14,8 @@
 
 # host address (domain name or IP address) the web UI can reach the worker by
 # (required by the web UI's developer mode and backends using jump-hosts to
-#  reach back to os-autoinst; set if the automatically deduced value is not
+#  reach back to os-autoinst; set to "auto" to enable auto-detection or
+#  to an explicit domain/address if the automatically deduced value is not
```

Comment:
so I don't know if I understand. If set to "auto" then WORKER_HOSTNAME will be populated with the FQDN. But if we don't specify the variable, like it is here with line 20 commented out, it will not be populated but the worker will still try to start just that the developer mode is likely not working, right?
If this is true, like for the openQA-in-openQA tests, I am not sure why we do this. Maybe you can clarify the effect better at least in the git commit message.

Reply:
> If set to "auto" then WORKER_HOSTNAME will be populated with the FQDN.

Right.

> But if we don't specify the variable, like it is here with line 20 commented out, it will not be populated but the worker will still try to start just that the developer mode is likely not working, right?

Right, like it was before we've been auto-initializing `WORKER_HOSTNAME`. Maybe I should double check whether it really doesn't make things worse for the developer mode.